src/WebJobs.Extensions.DocumentDB/Bindings/DocumentDBItemValueBinder.cs

```diff
@@ -16,21 +15,17 @@ namespace Microsoft.Azure.WebJobs.Extensions.DocumentDB
 {
     internal class DocumentDBItemValueBinder<T> : IValueBinder where T : class
     {
-        private ParameterInfo _parameter;
         private DocumentDBContext _context;
         private JObject _originalItem;
-        private DocumentDBItemBindingContext _itemBindingContext;
 
-        public DocumentDBItemValueBinder(ParameterInfo parameter, DocumentDBContext context, DocumentDBItemBindingContext itemBindingContext)
+        public DocumentDBItemValueBinder(DocumentDBContext context)
         {
-            _itemBindingContext = itemBindingContext;
-            _parameter = parameter;
             _context = context;
         }
 
         public Type Type
         {
-            get { return _parameter.ParameterType; }
```

Comment:
Do we have documentation about what this property is used for? I was following other patterns but I'm not sure where this gets called.

To answer your question -- yes, if you specify 'MyPoco' as the type you want, we'll construct a DocumentDBItemValueBinder<MyPoco>. I tried to think of a scenario where this wouldn't be the case, but couldn't. 

---
In reply to: [63894947](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/97#discussion_r63894947) [](ancestors = 63894947)